src/Stan/Inspection/AntiPattern.hs

```diff
@@ -388,3 +391,17 @@ stan0215 = mkAntiPatternInspection (Id "STAN-0215") "Slashes in paths" (FindAst
     pathLit :: PatternAst
     pathLit = PatternAstConstant (ContainStr "/")
         ||| PatternAstConstant (ContainStr "\\\\")
+
+-- | 'Inspection' - use 'Text' or 'ByteString'  instead of 'String' @STAN-0216@.
+stan0216 :: Inspection
+stan0216 = Inspection
+    { inspectionId = Id "STAN-0216"
+    , inspectionName = "String"
+    , inspectionDescription = "Usage of the default 'String' Haskell type"
+    , inspectionSolution =
+        [ "Consider using instead 'Text' for readable text, or 'ByteString' otherwise"
```

Comment:
I would split it into two separate suggestions 🙂 
To `ByteString` I would add instead of just otherwise, that it could be used for bytes or something like this. 
What you wrote is correct, and I would do the same at the beginning, but turns out people don't like when we talk about `ByteString` generally, so we need to be more careful in order not to trigger people and give them another reason "not to use Stan" 😄